modules/phase_field/src/materials/KKSPhaseConcentrationMaterial.C

```diff
@@ -45,11 +43,12 @@ KKSPhaseConcentrationMaterial::KKSPhaseConcentrationMaterial(const InputParamete
     _ci_old(_num_c * 2),
     _ci_IC(getParam<std::vector<Real>>("ci_IC")),
 
-    _f1(getMaterial("Fa_material")),
-    _f2(getMaterial("Fb_material")),
-    _Fi_names(getParam<std::vector<MaterialPropertyName>>("Fi_names")),
-    _prop_fi(2),
-    _fi_copy(2),
+    _F1(getMaterial("Fa_material")),
+    _F2(getMaterial("Fb_material")),
```

Comment:
The runtime error `To ensure dependency resolution, discrete materials must be retrieved during initial setup. This is a code problem.` is caused by these two lines. If I remove `_f1` and `_f2` entirely, that error is gone. However, I would get another runtime error `no MaterialPropertyName parameter named "Fa_material" found.`. We were using the MaterialBase and getMaterial the same way as before, strange that it is not working somehow. ("Fa_material" and "Fb_material" are defined as compute=false materials.) @dschwen